### PR TITLE
Add requirements to setup.py

### PR DESCRIPTION
**Pull request by MahdiZareie**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,6 +15,8 @@
                       "requests[security]",
                       "requests-toolbelt",
                       "beautifultable",
+                      "pytz",
+                      "tzlocal"
```
Comment (s-soroosh) [inline setup.py]: missing comma

Comment (MahdiZareie) [reply setup.py]: Done

File: setup.py [hunk 2]
```diff
@@ -15,6 +15,8 @@
                       "requests[security]",
                       "requests-toolbelt",
                       "beautifultable",
+                      "pytz",
+                      "tzlocal"و
```
Comment (s-soroosh) [inline setup.py]: Mahdiiiii

Comment (MahdiZareie) [reply setup.py]: :))

Comment (MahdiZareie) [reply setup.py]: sry, fixed

**Pull request merged**